test/integration/generated/bodyFileRest/README.md

```diff
@@ -0,0 +1,192 @@
+# Service client REST client library for JavaScript
+
+Test Infrastructure for AutoRest Swagger BAT
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@msinternal/body-file)
+- [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+
+## Getting started
+
+Our REST Clients provide simple, flexible, and reliable connections to REST APIs. Although these clients have raw access to the REST API of a service, we provide some functionality out-of-the-box such as Authentication with Azure Identity through our configurable REST [pipeline](https://github.com/Azure/azure-sdk-for-js/issues/8461). REST Clients also provide dev workflow benefits such as a strongly typed experience that helps to navigate the REST API through auto-complete and IntelliSense.
```

Comment:
I'm not sure we want to generate all this information all the time. I'd rather have a link to the wiki page that describes all this. With a link, it would be easier for us to fix any issues in the documentation or update it without having to re-generate all the packages.

Reply:
Updated